Frontend/src/components/CurrentLocation/CurrentLocation.jsx

```diff
@@ -0,0 +1,39 @@
+import currentLocation from '../../assets/currentLocation.svg'
+import './currentLocation.css'
+const CurrentLocation = () => {
+  const getCurrentCity = () => {
+    if (navigator.geolocation) {
+      navigator.geolocation.getCurrentPosition(successCallback, errorCallback);
+    } else {
+      console.log("Geolocation is not supported by this browser.");
+    }
+  };
+
+  const successCallback = (position) => {
+    const latitude = position.coords.latitude;
+    const longitude = position.coords.longitude;
+
+    fetch(`https://api.bigdatacloud.net/data/reverse-geocode-client?latitude=${latitude}&longitude=${longitude}&localityLanguage=en`)
+      .then(response => response.json())
+      .then(data => {
+        const city = data.city;
+        console.log(`Current city: ${city}`);
+      })
+      .catch(error => {
+        console.error("Error fetching city information:", error);
+      });
+  };
+
+  const errorCallback = (error) => {
+    console.error("Error getting location:", error.message);
+  };
+
+  return (
+    <div className='current-location-container'  onClick={getCurrentCity}>
+      <img src={currentLocation} alt="" className='image' />
+      <div className="text">Use current location</div>
```

Comment:
pls save all constants in a separate constants file similar to https://github.com/AYS-At-Your-Service/AYS/pull/13/files#diff-c2c392d453cc978fcfed88aae84cdeb1b2ce75f60d0bb32f26cc6e821ab39a7b

Reply:
done